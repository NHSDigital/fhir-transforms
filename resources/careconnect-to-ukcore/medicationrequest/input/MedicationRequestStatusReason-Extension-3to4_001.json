{
  "resourceType": "MedicationRequest",
  "extension": [
    {
      "url": "https://fhir.nhs.uk/STU3/StructureDefinition/Extension-CareConnect-GPC-MedicationStatusReason-1",
      "extension": [
        {
          "url": "statusReason",
          "valueCodeableConcept": {
            "coding": [
              {
                "system": "https://fhir.hl7.org.uk/Id/egton-codes",
                "code": "ALLERGY4",
                "display": "Adverse reaction to Central Nervous System Drugs",
                "userSelected": true
              },
              {
                "extension": [
                  {
                    "url": "https://fhir.nhs.uk/STU3/StructureDefinition/Extension-coding-sctdescid",
                    "extension": [
                      {
                        "url": "descriptionId",
                        "valueId": "1012851000006115"
                      },
                      {
                        "url": "descriptionDisplay",
                        "valueString": "Adverse reaction to Central Nervous System Drugs"
                      }
                    ]
                  }
                ],
                "system": "http://snomed.info/sct",
                "code": "1012851000006104",
                "display": "Adverse reaction to Central Nervous System Drugs"
              }
            ],
            "text": "Adverse reaction to Central Nervous System Drugs (allergy)"
          }
        },
        {
          "url": "statusChangeDate",
          "valueDateTime": "2011-09-28T10:22:32.423+01:00"
        }
      ]
    }
  ]
}
