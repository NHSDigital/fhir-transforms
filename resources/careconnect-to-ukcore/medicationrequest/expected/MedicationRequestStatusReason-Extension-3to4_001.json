{
  "resourceType": "MedicationRequest",
  "statusReason": {
    "coding": [
      {
        "system": "https://fhir.hl7.org.uk/Id/egton-codes",
        "code": "ALLERGY4",
        "display": "Adverse reaction to Central Nervous System Drugs",
        "userSelected": true
      },
      {
        "extension": [
          {
            "extension": [
              {
                "url": "descriptionId",
                "valueIdentifier": {
                  "system": "http://snomed.info/sct",
                  "value": "1012851000006115"
                }
              },
              {
                "url": "descriptionDisplay",
                "valueString": "Adverse reaction to Central Nervous System Drugs"
              }
            ],
            "url": "https://fhir.hl7.org.uk/StructureDefinition/Extension-UKCore-CodingSCTDescId"
          }
        ],
        "system": "http://snomed.info/sct",
        "code": "1012851000006104",
        "display": "Adverse reaction to Central Nervous System Drugs"
      }
    ],
    "text": "Adverse reaction to Central Nervous System Drugs (allergy)"
  }
}